src/jira/index.ts

```diff
@@ -0,0 +1,39 @@
+import JiraApi = require("jira-client");
+import * as TE from "fp-ts/lib/TaskEither";
+import * as A from "fp-ts/lib/ReadonlyArray";
+import { toError } from "fp-ts/lib/Either";
+import { errorsToReadableMessages } from "@pagopa/ts-commons/lib/reporters";
+import { pipe } from "fp-ts/lib/function";
+import { JiraIssueResponse } from "./types";
+
+export const jiraHostName = "pagopa.atlassian.net";
```

Comment:
```suggestion
export const JIRA_HOST_NAME = "pagopa.atlassian.net";
```

use uppercase for constants